src/main/scala/com/wavesplatform/Application.scala

```diff
@@ -16,8 +19,8 @@ import scorex.network.{TransactionalMessagesRepo, UnconfirmedPoolSynchronizer}
 import scorex.utils.ScorexLogging
 import scorex.waves.http.{DebugApiRoute, WavesApiRoute}
 import scorex.waves.transaction.WavesTransactionModule
-import BlockchainSettingsExtension._
 
+import scala.Option.option2Iterable
```

Comment:
There's no need to import this explicitly — implicit conversions are searched for in a source type's companion object by default.